packages/cactus-plugin-htlc-eth-besu-erc20/src/main/typescript/plugin-htlc-eth-besu-erc20.ts

```diff
@@ -52,6 +54,7 @@ export class PluginHtlcEthBesuErc20
     Checks.truthy(opts.pluginRegistry, `${fnTag} opts.pluginRegistry`);
     Checks.nonBlankString(opts.instanceId, `${fnTag} opts.instanceId`);
     this.instanceId = opts.instanceId;
+    this.keychainId = opts.keychainId || " ";
```

Comment:
@jagpreetsinghsasan Prefer default values that are easier to distinguish from non-default values. If it's an optional parameter then you could declare the field as an `Optional<string>` and then set it to `Optional.empty()` in the constructor if a `keychainId` was not provided in the ctor args.

Reply:
Is this fine?
![image](https://user-images.githubusercontent.com/56861721/136006588-093c805a-b48b-4add-88fa-6ac5a039166e.png)

Reply:
@jagpreetsinghsasan Looks legit, but wanted to ask: Why did you need the `.toString()` at the end?